xonsh/tools.py

```diff
@@ -633,17 +617,17 @@ def bool_seq_to_csv(x):
 
 
 def is_completions_display_value(x):
-    return x in {'none', 'single', 'multi'}
+    return x in ('none', 'single', 'multi')
```

Comment:
Yes. In your test you are including the time it takes to construct the object. It much faster to look stuff up in hash-tables than iterables.